examples/thermomechanics/thermal_deformation.cpp

```diff
@@ -81,8 +82,11 @@ void thermalDeformationExample( const std::string filename )
     // ====================================================
     //                    Force model
     // ====================================================
+    const double kappa = 1.0;
```

Comment:
I assume the values here for kappa and cp are arbitrary and need to be changed?

Reply:
Yes

Reply:
Sounds good.